pkg/resources/grant_ownership_acceptance_test.go

```diff
@@ -323,6 +323,108 @@ func TestAcc_GrantOwnership_OnObject_Table_ToDatabaseRole(t *testing.T) {
 	})
 }
 
+func TestAcc_GrantOwnership_OnObject_Procedure_ToAccountRole(t *testing.T) {
```

Comment:
nit: if next test has `WithoutArguments` part, it would be nice to have `WithArguments` here for consistent naming

Reply:
Done.